CI Docs failing to build

## Current Behavior
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->
```
Warning, treated as error:
/home/travis/build/pcdshub/happi/docs/source/client.rst:33:more than one target found for 'any' cross-reference 'Client': could be :py:class:`happi.Client` or :py:class:`happi.client.Client`
Makefile:24: recipe for target 'html' failed
make: *** [html] Error 2
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -31,8 +31,8 @@ env:
 
 jobs:
   allow_failures:
-    # This makes the PIP-based Python 3.6 optional.
-    - name: "Python 3.6 - PIP"
+    # This makes the PIP-based Python build optional.
+    - name: "Python - PIP"
 
 import:
   - pcdshub/pcds-ci-helpers:travis/shared_configs/setup-env-ui.yml

--- a/docs/source/client.rst
+++ b/docs/source/client.rst
@@ -31,7 +31,7 @@ information to be stored.
     db = JSONBackend(path='doc_test.json', initialize=True)
 
 If you are connecting to an existing database you can pass the information
-directly into the `Client` itself at ``__init__``. See :ref:`db_choice`
+directly into the :class:`.Client` itself at ``__init__``. See :ref:`db_choice`
 about how to configure your default backend choice.
 
 .. ipython:: python

